libs/natparse/src/main/java/org/amshove/natparse/lexing/SyntaxKind.java

```diff
@@ -333,6 +345,7 @@
 	DATA(true, false, false),
 	DATAAREA(true, false, false),
 	DATE(true, false, false),
+	DAT4E(true, false, false),
```

Comment:
This has to be `false, true, true` as it cannot be an identifier but is a system variable.
That way the failing test about keywords goes away.